app/mailers/chef_mailer.rb

```diff
@@ -3,7 +3,9 @@ class ChefMailer < ApplicationMailer
   
   def welcome_email
     @chef = params[:chef]
-    @url  = "http://localhost:3000/login?account_verified="+@chef.email
+    @url  = ENV['URL_AUTH']
```

Comment:
don't use this environment variable, instead use the url helper path in the email

Reply:
fixed.
The correct url helper is verify_url.